apps/web/src/design-system/dropdown/ThreeDot.tsx

```diff
@@ -0,0 +1,29 @@
+import useStyles from './Dropdown.styles';
+import React from 'react';
+export function ThreeDot({ ...props }) {
+  const { theme } = useStyles();
+
+  const dot = {
+    height: '7px',
+    width: '7px',
+    margin: '2px',
+    'background-color': theme.colorScheme == 'dark' ? '#525266' : '#525266',
+    'border-radius': '50%',
+    display: 'inline-block',
+    'font-weight': 'bold',
+  };
+
+  const threeDot = {
+    cursor: 'pointer',
+    padding: '5px',
+    margin: '5px',
+  };
+
+  return (
+    <div style={threeDot}>
+      <span style={dot}></span>
+      <span style={dot}></span>
+      <span style={dot}></span>
+    </div>
+  );
+}
```

Comment:
Sorry for missed this one, but we actually just have an SVG icon for the three dots so no need to recreate it. It located in `apps/web/src/design-system/icons/general/DotsHorizontal.tsx` So you can just import it instead of building it with CSS.

🙏

Reply:
done, sorry I didn't noticed